ui/src/vertical-geometry/plan-linking-header-item.tsx

```diff
@@ -60,11 +60,11 @@ export const PlanLinkingItemHeader: React.FC<PlanLinkingItemHeaderProps> = ({
                 width={width}
                 height={textLineTwoYPx + textDropAreaPx}>
                 <text
-                    className={styles['vertical-geometry-diagram__text-stroke-narrow']}
+                    className={styles['vertical-geometry-diagram__text-stroke-header']}
                     transform={`translate(0 ${textLineOneYPx}) scale(0.7)`}>
                     <tspan>{filename}</tspan>
                     <tspan x="0" dy={textLineTwoYPx}>
-                        {verticalCoordinateSystem},{' '}
+                        {verticalCoordinateSystem + ', '}
```

Comment:
Aiempi tapa piirsi epämukavan valkoisen taustan vieressä olevan tekstin päälle. Tällä tavalla rasterointimoottori käsittelee näitä samana tekstinä, eikä piirrä toisen outlinea toisen päälle.